### ISSUE TITLE
NPE in Utilities.isLanIpv4

### ISSUE THREAD

**Issue by ignoramous** [celzero/rethink-app#651]
```
Exception java.lang.NullPointerException:
  at com.celzero.bravedns.util.Utilities$Companion.isLanIpv4 (Utilities.kt:281)
  at com.celzero.bravedns.ui.SettingsFragment.showSocks5ProxyDialog$lambda-27 (SettingsFragment.kt:811)
  at com.celzero.bravedns.ui.SettingsFragment.$r8$lambda$VCkNW8fckio4lY2qHQ3NGNUwBLE (SettingsFragment.kt)
  at com.celzero.bravedns.ui.SettingsFragment$$InternalSyntheticLambda$2$a1791e42fe5c23058e41e63d1bda6be7f439062ebec089ab32eef8a801fdb378$0.onClick (SettingsFragment.java)
  at android.view.View.performClick (View.java:7758)
  at com.google.android.material.button.MaterialButton.performClick (MaterialButton.java:1194)
  at android.view.View.performClickInternal (View.java:7731)
  at android.view.View.access$3700 (View.java:862)
  at android.view.View$PerformClick.run (View.java:29338)
  at android.os.Handler.handleCallback (Handler.java:938)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loopOnce (Looper.java:210)
  at android.os.Looper.loop (Looper.java:299)
  at android.app.ActivityThread.main (ActivityThread.java:8268)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:576)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1073)
```

**Comment by ignoramous** [comment]
another:

```
Exception : java.lang.NullPointerException	Attempt to invoke virtual method 'boolean inet.ipaddr.IPAddress.isLoopback()' on a null object reference
  at com.celzero.bravedns.util.Utilities$Companion.isLanIpv4 (Utilities.kt:281)
  at com.celzero.bravedns.ui.DnsProxyListFragment.showAddDnsProxyDialog$lambda-2 (DnsProxyListFragment.kt:166)
  at com.celzero.bravedns.ui.DnsProxyListFragment.$r8$lambda$7w3IUcC9QzTg1NVPXQrVqyxnem4 (DnsProxyListFragment.kt)
  at com.celzero.bravedns.ui.DnsProxyListFragment$$InternalSyntheticLambda$2$bff19304853ded56cec6e88e04ad27079522765b8c65b51a5fc1c1d3cd5732d6$0.onClick (DnsProxyListFragment.java)
  at android.view.View.performClick (View.java:7505)
  at com.google.android.material.button.MaterialButton.performClick (MaterialButton.java:1194)
  at android.view.View.performClickInternal (View.java:7477)
  at android.view.View.access$3600 (View.java:824)
  at android.view.View$PerformClick.run (View.java:28662)
  at android.os.Handler.handleCallback (Handler.java:938)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:257)
  at android.app.ActivityThread.main (ActivityThread.java:8192)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:626)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1015)
```

**Comment by ignoramous** [comment]
from dns-proxy screen:

```
Exception java.lang.NullPointerException:
  at com.celzero.bravedns.util.Utilities$Companion.isLanIpv4 (Utilities.kt:281)
  at com.celzero.bravedns.ui.DnsProxyListFragment.showAddDnsProxyDialog$lambda-2 (DnsProxyListFragment.kt:166)
  at com.celzero.bravedns.ui.DnsProxyListFragment.$r8$lambda$7w3IUcC9QzTg1NVPXQrVqyxnem4 (DnsProxyListFragment.kt)
  at com.celzero.bravedns.ui.DnsProxyListFragment$$InternalSyntheticLambda$2$bff19304853ded56cec6e88e04ad27079522765b8c65b51a5fc1c1d3cd5732d6$0.onClick (DnsProxyListFragment.java)
  at android.view.View.performClick (View.java:7792)
  at android.widget.TextView.performClick (TextView.java:16112)
  at com.google.android.material.button.MaterialButton.performClick (MaterialButton.java:1194)
  at android.view.View.performClickInternal (View.java:7769)
  at android.view.View.access$3800 (View.java:910)
  at android.view.View$PerformClick.run (View.java:30218)
  at android.os.Handler.handleCallback (Handler.java:938)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loopOnce (Looper.java:226)
  at android.os.Looper.loop (Looper.java:313)
  at android.app.ActivityThread.main (ActivityThread.java:8751)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:571)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1135)
```

**Comment by hussainmohd-a** [comment]
Issue got fixed earlier: b55671ddd87b101ce6f028342cc37072d278efac
